app/migrations/Version20180529102348.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20180529102348 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        $this->addSql('ALTER TABLE donation_transactions ADD type VARCHAR(255) NOT NULL DEFAULT \'cb\'');
```

Comment:
Not a big fan of having a default value on the column.
It's fragile.
If for whatever reason we try to insert a new line in this table it would work while we potentially have a bug in the codebase.

Reply:
That's why I alter it with an other change to remove default value ^^,

Reply:
My bad, thank you for pointing this out ;-)